backend/local/local.go

```diff
@@ -488,41 +498,20 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 		if useReadDir {
 			// Windows and Plan9 read the directory entries with the stat information in which
 			// shouldn't fail because of unreadable entries.
-			fis, err = fd.Readdir(1024)
+			fis, err = fd.Readdir(dirListingBatchSize)
 			if err == io.EOF && len(fis) == 0 {
 				break
 			}
 		} else {
 			// For other OSes we read the names only (which shouldn't fail) then stat the
-			// individual ourselves so we can log errors but not fail the directory read.
+			// individual ourselves, so we can log errors but not fail the directory read.
 			var names []string
-			names, err = fd.Readdirnames(1024)
+			names, err = fd.Readdirnames(dirListingBatchSize)
 			if err == io.EOF && len(names) == 0 {
 				break
 			}
 			if err == nil {
-				for _, name := range names {
```

Comment:
I am not sure whether we need to do the whole for loop in parallel or just the 1024 entries batch at once in parallel... Open to suggestions.